OpenKeychain/src/main/res/xml/sync_preferences.xml

```diff
@@ -4,6 +4,11 @@
         android:persistent="false"
         android:title="@string/label_sync_settings_keyserver_title"/>
     <SwitchPreference
+        android:key="enableWifiSyncOnly"
+        android:persistent="true"
+        android:dependency="syncKeyserver"
```

Comment:
Oops, sorry for the delay. Thought I had mentioned this, but I hadn't. `dependency` shouldn't work since we don't use the keyserver sync preference directly, did you test this?